adafruit_vcnl4010.py

```diff
@@ -30,6 +30,11 @@
 
 from adafruit_bus_device import i2c_device
 
+try:
+    from typing import Optional, List  # pylint: disable=unused-import
```

Comment:
The pylint disable isn't needed since the typing module is being used.

Reply:
Since the imported items aren't being used:

```suggestion
    import typing  # pylint: disable=unused-import
```